### ISSUE TITLE
 Watchers: fail to list watchers, TypeError: this._$http is not a function

### ISSUE THREAD

**Issue by hAtvis** [sirensolutions/sentinl#380]
1. Sentinl and Kibana version 
5.6.4
2. Issue description
after i installed sentinl, i restart kibana. when i open sentinl, it shows me 
Watchers: fail to list watchers, TypeError: this._$http is not a function

3. Reproducing the issue step-by-step
i install sentinl with:
/usr/share/kibana/bin/kibana-plugin install https://github.com/sirensolutions/sentinl/releases/download/tag-5.6.2/sentinl-v5.6.4.zip

then i restart kibana with:
/etc/init.d/kibana restart

4. Expected results vs. actual results

5. Sample of data

6. Additional details

**Comment by lmangani** [comment]
@hAtvis this is a Kibana failure, you will need to tell us everything relevant about your setup here and how the installation went to even attempt to help. For instance, do you have authentication, or any other plugins installed?

**Comment by shizhoulong356** [comment]
With the same questions:
I also install  kibana through the commands: yum  -y install  kibana-5.6.4-x86_64.rpm 

and this is my configuraiton of kibana:  
server.port: 5601
server.host: "0.0.0.0"
elasticsearch.url: "http://10.2.34.129:9200"
kibana.index: ".kibana"
there is nothing sepcial


install the pligins:
./bin/kibana-plugin install https://github.com/sirensolutions/sentinl/releases/download/tag-5.6.2/sentinl-v5.6.4.zip

but:
[root@s-xiasha-10-2-34-131 kibana]# ./bin/kibana-plugin  list
sentinl@10.0.0-SNAPSHOT

then restart the kibana:  /etc/init.d/kibana restart

then i open my kibana:
the sentinl ui prompts:
Watchers: fail to list watchers, TypeError: this._$http is not a function
Watchers: fail to list scripts, TypeError: this._$http is not a function



so what's the problem?  Does the source code is the wrong version

**Comment by lmangani** [comment]
@hAtvis I have no familiarity with the package, could you confirm if you are installing the open-source version of Kibana, or the one riddled with the Elastic commercial extensions?

**Comment by shizhoulong356** [comment]
@lmangani ,Nope,  thereis no other extensions.
We all download the kibana package through the official sites

**Comment by hAtvis** [comment]
kibana was downloaded from https://artifacts.elastic.co/downloads/kibana/kibana-5.6.4-x86_64.rpm
I just confirmed that sha512 is the same

**Comment by lmangani** [comment]
Could you guys confirm if the Kibana you are installing comes with x-pack installed? I'm not sure if this is the case for packages, but for containers it sure is:
```
The images are available in two different configurations or "flavors". The x-pack flavor, which is the default, ships with X-Pack features pre-installed. The oss flavor does not include X-Pack, and contains only open source Kibana.
```

**Comment by shizhoulong356** [comment]
we not running kibana on docker,  but on the physical machine based on centos 6.5 .  So We're sure that x-pack is not installed at all .  And  our kibana version is 5.6.4

**Comment by sergibondarenko** [comment]
@hAtvis I can reproduce this. There will be a fix soon

**Comment by sergibondarenko** [comment]
@hAtvis @shizhoulong356 guys, it is v5.6.x bug only. Do you have an option to use Kibana v6+?

**Comment by shizhoulong356** [comment]
But Our All application Components are base on 5.6.4 , If Kibana  v6+ has no problems, we can try

**Comment by shizhoulong356** [comment]
@hAtvis  We tried v6+, but my kibana  can't compationable with my Elasticsearch. So  Look forward for your bug fix

**Comment by sergibondarenko** [comment]
@hAtvis @shizhoulong356 guys, meanwhile use the following temporary workaround 
1. Download Sentinl from here https://github.com/sirensolutions/sentinl-private/archive/tag-5.6.2.tar.gz
2. Unpack it `tar -xzvf sentinl-private-tag-5.6.2.tar.gz`
3. Go inside `cd sentinl-private-tag-5.6.2`
4. Change `version` from `5.6.2` to `5.6.4` in the `packages.json`
5. Install libs `npm install && npm install -g gulp`
6. Install Sentinl `gulp dev --kibanahomepath=/path/to/kibana`
Press `Ctrl+c` on your keyboard when you see the message like the following one
```
[10:46:07] Finished 'sync' after 1.03 s
[10:46:07] Starting 'dev'...
```
7. Restart Kibana and use Sentinl

**Comment by szydan** [comment]
@sergibondarenko Instead of 
```
gulp dev --kibanahomepath=/path/to/kibana
```

You can run 
```
gulp sync --kibanahomepath=/path/to/kibana
```

Then there is no need to press Ctrl+C

**Comment by shizhoulong356** [comment]
I can't download the private version  As  you mentioned above,   the return code is 404 @sergibondarenko

**Comment by sergibondarenko** [comment]
@shizhoulong356 @hAtvis 
Added archives https://github.com/sirensolutions/sentinl/releases/tag/tag-5.6.2

**Comment by danielmotaleite** [comment]
Was the fixed applyed to the branch-5.6 ? or the only fixed version is the above tag?

**Comment by lmangani** [comment]
Closing for inactivity, feel free to comment to reopen discussion anytime.
